### PR TITLE
[Review] Request from 'schubi2' @ 'yast/yast-storage/review_160302_efi_boot_formating'

### PR DESCRIPTION
**Pull request by schubi2**
Please review the following changes:
  * 0bc6862 packaging
  * 2b7cb47 Efi boot: Select Efi Boot file system ID and allowing formating at the same time

### CODE REVIEW

File: src/include/partitioning/ep-dialogs.rb [hunk 1]
```diff
@@ -48,6 +48,15 @@ def MiniWorkflowStepRole(data)
 
       role = data.value.fetch("role", :data)
 
+      efi_select = Empty()
+      if Partitions.EfiBoot && (Arch.i386||Arch.x86_64)
```
Comment (aschnell) [inline src/include/partitioning/ep-dialogs.rb]: ARM is also EFI.

File: package/yast2-storage.changes [hunk 2]
```diff
@@ -17,6 +24,7 @@ Mon Mar  7 16:17:03 UTC 2016 - ancor@suse.com
 
 - Removed support for flexible partitioning proposal, a feature
   not used in SLE or openSUSE since years (fate#320111)
+>>>>>>> master
```
Comment (aschnell) [inline package/yast2-storage.changes]: Hmm.

**Comment by aschnell** [comment]
Do I see it right that you can only create a EFI Boot Partition but not change an existing to an EFI Boot Partition? Is that clear to the usability experts?

File: src/include/partitioning/ep-dialogs.rb [hunk 3]
```diff
@@ -441,6 +464,14 @@ def MiniWorkflowStepFormatMount(orig_data, creating: false)
       UI.ChangeWidget(Id(:do_mount_attachment), :Enabled, do_mount)
       UI.ChangeWidget(Id(:mount_point), :Value, mount)
 
+      if role == :efi_boot
```
Comment (schubi2) [inline src/include/partitioning/ep-dialogs.rb]: As far I see role will be set only for new created partitions (in MiniWorkflowStepRole). So while the update workflow these entries should not be disabled.

**Comment by schubi2** [comment]
So this hits new generated partitions only. This has been approved by the UI experts.

**Comment by aschnell** [comment]
OK. LGTM.

**Comment by schubi2** [comment]
Thanks !

**Pull request merged**